configuration/777f9/genesis.json

```diff
@@ -72,7 +72,7 @@
             "33337aeeaddce8232018ab9397bbe7301fe7a533fc57c91005ea3dc42df2a9ca3e439d6f3b69b551d5642035efbe77fe1af102530448e7cf677571b1bafef903"
         }
     }
-, "startTime": 1572966898
+, "startTime": 1575051828
```

Comment:
Is this for a real pbft genesis?

If so I think we cannot use fixed pre-generated genesis files since the start time matters. Nodes have to be started and create blocks within K slots (or 2K I forget) of the start time.

Reply:
So we need to generate them at the start of each script then. The issue there is that `GenesisHash` is in the `config.yaml` file so I'll need to add a `--genesis-hash` parser to the cli. Does that sound reasonable?

Reply:
I'll remove the pre-generated files and modify the scripts as described above.